### PR TITLE
Adds basic tests for all main meshes.

### LINKED ISSUE #22: Add tests for new code
I am trying to get codecov enabled on PRs - until then, we should add some basic tests (not figure tests) just to make sure code is working, and to get coverage back up to 100% - currently it is ~50%: https://app.codecov.io/gh/sunpy/sunkit-pyvista/

### PR DESCRIPTION
**Pull request by jeffreypaul15**
Fixes #22 

Tests that increase the codecov of the project, we still need to decide how to test meshes. 
As of now, I'm testing each individual mesh with it's `n_cells` and `n_points` value.

- [ ] Test for Pfsspy field lines
Based on how #21 goes, we need to decide whether to access each field line separately as they are added together and there isn't any way to get all the field lines and check their number of cells/points.

### CODE REVIEW

File: sunkit_pyvista/plotter.py [hunk 1]
```diff
@@ -127,9 +127,9 @@ def plot_map(self, m, **kwargs):
         **kwargs :
             Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
         """
-        cmap = kwargs.pop('cmap', m.cmap)
+        kwargs.pop('cmap', m.cmap)
```
Comment (nabobalis) [inline sunkit_pyvista/plotter.py:130]: What was the reason for this change?

Comment (jeffreypaul15) [reply sunkit_pyvista/plotter.py:130]: My bad, that wasn't supposed to be a thing. I did that to demonstrate how a plot without cmap would look.

**Comment by jeffreypaul15** [comment]
Do we need a changelog for tests?

**Comment by nabobalis** [comment]
Sure add a trivial one.

**Comment by dstansby** [comment]
I'm just going to close and open this to see if codecov responds.

**Comment by dstansby** [comment]
Ah excellent, +52% coverage isn't bad 😄

**Review by dstansby — APPROVED** [review]
👍 nice work

**Pull request merged**